src/Components/Search/SearchBar.tsx

```diff
@@ -410,7 +412,7 @@ export class SearchBar extends Component<Props, State> {
     // @ts-expect-error PLEASE_FIX_ME_STRICT_NULL_CHECK_MIGRATION
     const edges = get(viewer, v => v.searchConnection.edges, [])
     // @ts-expect-error PLEASE_FIX_ME_STRICT_NULL_CHECK_MIGRATION
-    const suggestions = [firstSuggestionPlaceholder, ...edges]
+    const suggestions = [...edges, firstSuggestionPlaceholder]
```

Comment:
**question**: What do you think about renaming `firstSuggestionPlaceholder`? For example, in `lastSuggestionPlaceholder` or even better in `seeAllResultsPlaceholder`

Reply:
Also, I would rename all the related code (for example, as [here](https://github.com/artsy/force/blob/056073747177a3b77a2dd5bbbb068760dbb9b0ae/src/Components/Search/SearchBar.tsx#L341), [here](https://github.com/artsy/force/blob/056073747177a3b77a2dd5bbbb068760dbb9b0ae/src/Components/Search/SearchBar.tsx#L408) and [here](https://github.com/artsy/force/blob/056073747177a3b77a2dd5bbbb068760dbb9b0ae/src/Components/Search/Suggestions/SuggestionItem.tsx#L32))

Reply:
Good idea